### PR TITLE
[TR] PIM-4468: add uncheck feature

### PR DESCRIPTION
**Pull request by sumbobyboys**

### CODE REVIEW

File: features/Context/DataGridContext.php [hunk 1]
```diff
@@ -625,6 +625,29 @@ public function iCheckTheRows($rows)
      *
      * @throws ExpectationException
      *
+     * @When /^I uncheck the rows? "([^"]*)"$/
+     */
+    public function iUnheckTheRows($rows)
```
Comment (jmleroux) [inline features/Context/DataGridContext.php]: iUncheckTheRows

File: features/Context/DataGridContext.php [hunk 2]
```diff
@@ -625,6 +625,29 @@ public function iCheckTheRows($rows)
      *
      * @throws ExpectationException
      *
+     * @When /^I uncheck the rows? "([^"]*)"$/
```
Comment (jmleroux) [inline features/Context/DataGridContext.php]: `@When /^I uncheck the rows? "([^"]+)"$/`

Comment (sumbobyboys) [inline features/Context/DataGridContext.php]: I used the same as above (check the row). But yeah, you are right

Comment (jmleroux) [inline features/Context/DataGridContext.php]: We use a lot of `"([^"]*)"` (it will match "") in the features when `([^"]+)` is more accurate imho.

**Comment by jmleroux** [comment]
GTM after fixes

**Pull request merged**